app/views/content/is-teaching-right-for-me/maths/maths/_article.html.erb

```diff
@@ -0,0 +1,95 @@
+    <p>You may also get the opportunity to teach other subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on individual schools.
+    </div>
+</section>
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my pupils",
+          text: "I enjoy showing my pupils that maths is essential in everyday life. Whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
+          accreditation: "Rotna Roy, Maths Lead at a school in London",
+        }
+      ],
+      colors: %w[yellow],
+      numbered: false
+    ) %>
+  </section>
+
+<section class="col col-space-m col-720">
+<h2>Fund your teacher training</h2>
+<p>To train to be a maths teacher, you will need to do a teacher training course. </p>
+<p>Tax-free bursaries of £28,000 and scholarships of £30,000 are available for <a href="/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=maths&button=#funding-widget">eligible trainee maths teachers</a>. </p>
+<p>You can find out more about <a href="https://teachingmathsscholars.org/home">the maths scholarship from The Institute of Mathematics and its Applications, and partners</a>.</p>
+<p>You can receive this alongside a tuition fee and maintenance loan. </p>
+<p>You may also be able to get extra funding support if you are: </p>
+<ul>
+<li>a parent or carer </li>
+<li>disabled </li>
+</ul>
+<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
+<h2>Check your qualifications</h2>
+<p>You'll need a bachelors degree before training to teach, but it does not have to be in maths.</p>
+<p>Besides maths, the most popular degrees people have before doing a teacher training course in maths are:</p>
+<ul>
+<li>economics </li>
+<li>psychology </li>
+<li>accounting </li>
+<li>mechanical engineering </li>
+</ul>
+<p>To train to teach in primary and secondary schools in England, you’ll need: </p>
+<ul>
+<li>GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary) </li>
+<li>a bachelor’s degree in any subject </li>
+</ul>
+<p>Your training provider may need you to brush up on your computing knowledge before you become a maths teacher. You can do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE) course</a> before you start your teacher training to top up your subject knowledge. </p>
+<p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out more about the qualifications needed to teach</a>.</p>
+</section>
+</div>
```

Comment:
The page in prod has info on internships (and get school experience) - shouldn't we still include this?

Reply:
@gemmadallmandfe I don't necessarily think we still need to include it as we're talking about subjects and qualifications then directing people to get an adviser. So it feels a bit forced to me to have internships and school experience in here too as it's not quite the same thing?

Reply:
We do have UR insight that people wanted this internship info on the subject page - will share